data/card_catalogue/unique-1.json

```diff
@@ -73,7 +73,7 @@
         "active": "Cast a handful of sand and pocket lint into the air before you.",
         "passive": null
       },
-      "flavour": "Good for blinding a close range target.",
+      "flavour": "In total war, all tactics are acceptable.",
```

Comment:
I can't make a PR without a bunch of unrelated flavour text changes I forgot about